doc/src/manual/performance-tips.md

```diff
@@ -1507,3 +1507,90 @@ The following examples may help you interpret expressions marked as containing n
         field `data::Array{T}`. But `Array` needs the dimension `N`, too, to be a concrete type.
       * Suggestion: use concrete types like `Array{T,3}` or `Array{T,N}`, where `N` is now a parameter
         of `ArrayContainer`
+
+## [Performance of captured variable](@id man-performance-captured)
+
+Consider the following example that defines two inner functions:
+```
+    function abmult(r::Int)
```

Comment:
Code blocks in markdown shouldn't be indented a level if they're inside triple backticks, so this should be deindented. Also the backticks should have a `julia` annotation.